packages/synapse-react-client/src/components/AccessRequirementList/AccessApprovalCheckMark.tsx

```diff
@@ -6,8 +6,7 @@ import {
 } from '@mui/icons-material'
 import Skeleton from '@mui/material/Skeleton'
 import ConditionalWrapper from '../utils/ConditionalWrapper'
-import { Avatar, AvatarProps, styled, SxProps, useTheme } from '@mui/material'
-import { StyledComponent } from '@emotion/styled/dist/emotion-styled.cjs'
```

Comment:
`dist/emotion-styled.cjs` is not exported by `@emotion/styled`, so just remove this type